test/README.md

```diff
@@ -0,0 +1,78 @@
+# HOW TO USE IT
+
+This document describes how to use the benchmarking scripts.
+
+Those tests are an example of benchmarking tests using bench and htcmock to measure the performances of ArmoniK.  
+We have to deploy [ArmoniK(aws-benchmark)](https://github.com/aneoconsulting/ArmoniK/tree/yk/benchmarking_scripts/infrastructure/quick-deploy/aws-benchmark) on aws with two partitions (bench and htcmock) with 100 pods for each partition
```

Comment:
You can't put a link to a branch on your documentation since the branch will be deleted when the PR will be merged.

Reply:
done !